CONTRIBUTING.md

```diff
@@ -22,21 +57,55 @@ the Sonatype OSS snapshots repository at `https://oss.sonatype.org/content/repos
 Building using Java 11+:
 
 ```bash
-java -version
+$ java -version
 ```
 
 ```bash
-./gradlew assemble
+$ ./gradlew assemble
+
+# Build the complete project
+$ ./gradlew build
 ```
 
-### Style guide
+## Testing
+
+```bash
+# Run integration tests
+$ ./gradlew integrationTest
+```
+
+### Test Types:
+
+- Unit Tests: Isolated component tests.
+
+- Integration Tests: Validate interactions between modules.
+
+- CI Checks: PRs must pass all tests and linting.
 
-See
-the [Style guide](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md)
-from the opentelemetry-java-instrumentation repository.
+Failure? Check logs for errors or mismatched dependencies.
+
+
+## Contributing Rules
+
+Style Guide: Follow the Java Instrumentation Style Guide.
```

Comment:
you removed the link to the style guide in the other repo, which should be added back